Fix documentation build

### Situation
By the time writing this the documentation build fails because of a not answered domain message:

```java
Exception in thread "main" java.io.IOException: Generation of 'messaging-files (needs integration tests)' failed:NOT VALID:message with id RESULT_DETAILS_FOR_JOB_FINDING is send sync, but nobody answers it !?!?
        at com.mercedesbenz.sechub.docgen.AsciidocGenerator$AsciidocGeneratorExecutor.handleError(AsciidocGenerator.java:190)
        at com.mercedesbenz.sechub.docgen.AsciidocGenerator$AsciidocGeneratorExecutor.execute(AsciidocGenerator.java:177)
        at com.mercedesbenz.sechub.docgen.AsciidocGenerator.generate(AsciidocGenerator.java:148)
        at com.mercedesbenz.sechub.docgen.AsciidocGenerator.main(AsciidocGenerator.java:89)
Caused by: java.lang.IllegalStateException: NOT VALID:message with id RESULT_DETAILS_FOR_JOB_FINDING is send sync, but nobody answers it !?!?
        at com.mercedesbenz.sechub.docgen.messaging.DomainMessagingModelPlantUMLGenerator.handleValidationError(DomainMessagingModelPlantUMLGenerator.java:228)
```

### Solution
Analyze the problem and add the missing call to make the documentation build work again.

--- a/sechub-assistant/src/main/java/com/mercedesbenz/sechub/assistant/SecHubExplanationInputCollector.java
+++ b/sechub-assistant/src/main/java/com/mercedesbenz/sechub/assistant/SecHubExplanationInputCollector.java
@@ -10,6 +10,7 @@ import com.mercedesbenz.sechub.sharedkernel.messaging.DomainMessageService;
 import com.mercedesbenz.sechub.sharedkernel.messaging.DomainMessageSynchronousResult;
 import com.mercedesbenz.sechub.sharedkernel.messaging.IsRecevingSyncMessage;
 import com.mercedesbenz.sechub.sharedkernel.messaging.IsSendingSyncMessage;
+import com.mercedesbenz.sechub.sharedkernel.messaging.IsSendingSyncMessageAnswer;
 import com.mercedesbenz.sechub.sharedkernel.messaging.JobFinding;
 import com.mercedesbenz.sechub.sharedkernel.messaging.MessageDataKeys;
 import com.mercedesbenz.sechub.sharedkernel.messaging.MessageID;
@@ -25,6 +26,7 @@ public class SecHubExplanationInputCollector {
 
     @IsSendingSyncMessage(MessageID.REQUEST_DETAILS_FOR_JOB_FINDING)
     @IsRecevingSyncMessage(MessageID.RESULT_DETAILS_FOR_JOB_FINDING)
+    @IsSendingSyncMessageAnswer(value = MessageID.REQUEST_DETAILS_FOR_JOB_FINDING, answeringTo = MessageID.RESULT_DETAILS_FOR_JOB_FINDING, branchName = "success")
     public SecHubExplanationInput collectInput(String projectId, UUID jobUUID, int findingId) {
 
         DomainMessage domainMessage = new DomainMessage(MessageID.REQUEST_DETAILS_FOR_JOB_FINDING);

